docs/modules/ROOT/pages/reference/extensions/mybatis.adoc

```diff
@@ -34,3 +38,25 @@ Please refer to the above links for usage and configuration details.
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Additional Camel Quarkus configuration
+
+Please refer to https://quarkiverse.github.io/quarkiverse-docs/quarkus-mybatis/dev/index.html[Quarkus MyBatis] for configuration. It must enable the following options
+
+[source, properties]
+----
+quarkus.mybatis.xmlconfig.enable=true
+quarkus.mybatis.xmlconfig.path=SqlMapConfig.xml
+----
+TIP: `quarkus.mybatis.xmlconfig.path` must be the same with `configurationUri` param in the mybatis endpoint.
+
+If you want to run in the native mode, you have to set these properties in pom.xml currently.
+
+[source, xml]
+----
+<properties>
+    <quarkus.package.type>native</quarkus.package.type>
```

Comment:
`<quarkus.package.type>native</quarkus.package.type>` is not MyBatis specific and it does not need to be documented here.